src/gui/guiScene.cpp

```diff
@@ -32,8 +32,6 @@ GUIScene::GUIScene(gui::IGUIEnvironment *env, scene::ISceneManager *smgr,
 
 	m_cam = m_smgr->addCameraSceneNode(0, v3f(0.f, 0.f, -100.f), v3f(0.f));
 	m_cam->setFOV(30.f * core::DEGTORAD);
-
-	m_smgr->getParameters()->setAttribute(scene::ALLOW_ZWRITE_ON_TRANSPARENT, true);
```

Comment:
hmm actually if these have to be hardcoded inside IrrlichtMt now that's worse IMO
guess `IAttributes` has to be kept then

Reply:
It would never be false anyways though?

Reply:
It removes the flexibility of being able to change these in the future if that's ever required.

Reply:
The setAllowZWriteOnTransparent() method itself in IVideoDriver (which is exposed to Minetest I think?) would probably be better to use if that need ever arises. And we don't use OBJ .mtl files but if we ever do, I'd assume it wouldn't be as easy as flipping a switch in Minetest anyways.

Reply:
`setAllowZWriteOnTransparent` exists but it cannot be used to influence the behavior of `CSceneManager::drawAll()` anymore:
![grafik](https://user-images.githubusercontent.com/1042418/147656136-47ad7269-c45e-4f0e-ba3e-151ee7b74e7c.png)